habitat-lab/habitat/config/CONFIG_KEYS.md

```diff
@@ -0,0 +1,180 @@
+# Configuration Keys
+
+## Dataset
+Configuration for the dataset of the task.
+A dataset consists of episodes
+(a start configuration for a task within a scene) and a scene dataset
+(with all the assets needed to instantiate the task)
+
+| Key | Description |
+| --- | --- |
+| habitat.dataset.type |  The key for the dataset class that will be used. Examples of such keys are `PointNav-v1`, `ObjectNav-v1`, `InstanceImageNav-v1` or `RearrangeDataset-v0`. Different datasets have different properties so you should use the dataset that fits your task. |
+| habitat.dataset.scene_dir | The path to the directory containing the scenes that will be used. You should put all your scenes in the same folder (example `data/scene_datasets`) to avoid having to change it. |
+|habitat.dataset.data_path | The path to the episode dataset. Episodes need to be compatible with the `type` argument (so they will load properly) and only use scenes that are present in the `scenes_dir`.|
+|habitat.dataset.split | `data_path` can have a `split` in the path. For example: "data/datasets/pointnav/habitat-test-scenes/v1/{split}/{split}.json.gz" the value in "{split}" will be replaced by the value of the `split` argument. This allows to easily swap between `train` and `eval` episodes by only changing the split argument.|
+
+## Task
+The definition of the task in Habitat.
+There are many different Tasks determined by the `habitat.task.type` config:
+ - Point navigation : `Nav-0`
+ - Image navigation : `Nav-0`
```

Comment:
Both [pointnav](https://github.com/facebookresearch/habitat-lab/blob/main/habitat-lab/habitat/config/habitat/task/pointnav.yaml) and [imagenav](https://github.com/facebookresearch/habitat-lab/blob/main/habitat-lab/habitat/config/habitat/task/imagenav.yaml) use Nav-0 for their task. I guess task got reused because the two are not that different.